x-pack/plugins/apm/public/components/shared/TransactionActionMenu/sections.ts

```diff
@@ -62,7 +62,7 @@ export const getSections = ({
 
   const uptimeLink = url.format({
     pathname: basePath.prepend('/app/uptime'),
-    hash: `/?${fromQuery(
+    search: `?${fromQuery(
```

Comment:
Oh, was this the actual bug?

Reply:
FWICT both `hash` and `search` work when linking to Uptime. But using hash somehow results in a double `#` when using `navigateToApp`.